### PR TITLE
Show closed text on closed auctions

### PR DESCRIPTION
**Pull request by adelevie**

### CODE REVIEW

**Comment by harrisj** [comment]
I also added a label/test for Expiring and Coming Soon auctions. Text tweaks welcome. Note that all of them use the same color labels for now because the Federal Web Design Standards support any label color as long as it's blue. I expect @ddotrollins will have some good advice, but this can wait until 2016. Thanks!

**Comment by harrisj** [comment]
I'm not proud of using `!important` in here, but sometimes I do things I'm ashamed of

File: spec/features/bidder_interacts_with_auction_spec.rb [hunk 1]
```diff
@@ -2,153 +2,205 @@
...
-    click_on("Submit")
-
-    # returns us back to the bid page
-    expect(page).to have_content("Current bid:")
-    expect(page).to have_content("$800.00")
+      expect(page).to_not have_content('Bid »')
+    end
+
+    scenario "There are several auctions" do
+      # Test ordering here
```
Comment (adelevie) [inline spec/features/bidder_interacts_with_auction_spec.rb]: @harrisj should this be implemented? Or could this be skipped and included in a subsequent PR?

Comment (harrisj) [inline spec/features/bidder_interacts_with_auction_spec.rb]: Oh, we could do it on subsequent PRs (I wanted to do it after ordering is done)

Comment (adelevie) [inline spec/features/bidder_interacts_with_auction_spec.rb]: Sounds good. Is there a way to mark this as skipped?

Comment (harrisj) [inline spec/features/bidder_interacts_with_auction_spec.rb]: I think it is automatically skipped and marked if it is empty?

Comment (adelevie) [inline spec/features/bidder_interacts_with_auction_spec.rb]: :+1:

**Pull request merged**